jina/types/arrays/memmap.py

```diff
@@ -147,7 +147,7 @@ def append(self, doc: 'Document', flush: bool = True) -> None:
             ).tobytes()
         )
         self._header_map[doc.id] = (len(self._header_map), p, r, r + l)
-        self._start = p + r + l
+        self._start = p + r + l  # <==> self._start = self._start + l
```

Comment:
can't understand the comment here, suggestion: either delete it or write it clearly

```suggestion
        self._start = p + r + l
```